wcomponents-core/src/main/resources/schema/ui/v1/fieldIndicator.xsd

```diff
@@ -3,7 +3,7 @@
 	xmlns:ui="https://github.com/bordertech/wcomponents/namespace/ui/v1.0">
 	<xs:element name="fieldindicator">
```

Comment:
Found the mixed camel case of fieldIndicator.xsd and ui:fileindicator a bit distracting at first, but is OK.

Reply:
it is an artefact of the old WComponents schema which used camel case element names. Git is file-name case insensitive so it would be annoying to rename them.